src/Databases/DatabaseOrdinary.cpp

```diff
@@ -200,10 +207,14 @@ void DatabaseOrdinary::startupTables(ThreadPool & thread_pool)
     AtomicStopwatch watch;
     std::atomic<size_t> tables_processed{0};
 
-    auto startup_one_table = [&](const StoragePtr & table) {
+    // clang-format off
```

Comment:
Looks like clang-format doesn't respect brace wrapping for lambdas. Looks like it passes the style checks now:
https://clickhouse-test-reports.s3.yandex.net/11900/1244ca562759c4b92a08f4350080cc01567e8c99/style_check.html

Before it failed: https://clickhouse-test-reports.s3.yandex.net/11900/ea40e7b4039f69e8e1cb20ce23039ed0c93bc7ca/style_check.html#fail1